CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Model/Handbook.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Handbook.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-11.
+//
+
+import SwiftUI
+
+struct Handbook: Identifiable {
+    let id = UUID()
+    var title: String
+    var subtitle: String
+    var text: String
+    var logo: String
+    var image: String
+    var color1: Color
```

Comment:
what is the difference between color1 and color2 ?

Reply:
The 2 colors are used to make a linear gradient color for the background of the handbook component at the bottom of this view. Do you think there should be a better way to name them?

![Simulator Screen Shot - iPhone 14 Pro - 2022-11-15 at 12 11 01](https://user-images.githubusercontent.com/44596576/201983648-9dcbdabe-3a9a-4f92-ba23-4717385f3bc8.png)

Reply:
@sghourab yes, it should be colorLeft/right, colorTop/bottom depends on the direction.